lab_solutions/lab03_3_rational_number_class.py

```diff
@@ -125,6 +126,10 @@ def main():
     assert r1 * r2 == RationalNumber(12, 6)
     assert r1 / r2 == RationalNumber(9, 8)
     assert r1 == RationalNumber(6, 4)
+    r4 = RationalNumber(2, 3)
+    r4.p = 1100
+    r4.q = 55
```

Comment:
You could make this test more explicit with:
```suggestion
    r4.p = 20 * 55
    r4.q = 55
```